File: app/src/main/java/com/plain/awesome_clock_ace/utils/DateUtils.java
package com.plain.awesome_clock_ace.utils;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

public class DateUtils {

    public static String getHour() {
        Date date = new Date();
        SimpleDateFormat dateFormat = new SimpleDateFormat("HH", Locale.getDefault());
        return dateFormat.format(date);
    }

}

File: app/src/main/java/com/plain/awesome_clock_ace/utils/DateUtils.kt
package com.plain.awesome_clock_ace.utils

import java.text.SimpleDateFormat
import java.util.*

/**
 *  name: DateUtils
 *  desc: TODO
 *  date: 2020/3/16 4:53 PM
 *  version: v1.0
 *  author: Plain
 *  blog: https://plain-dev.com
 *  email: support@plain-dev.com
 */
object DateUtils {

    fun getHour(): String {
        val date = Date()
        val dateFormat = SimpleDateFormat("HH", Locale.getDefault())
        return dateFormat.format(date)
    }

}